app/assets/javascripts/applications.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Lets not track these files into the git repo:
`git rm app/assets/javascripts/applications.coffee`